docs/resources/confluent_subject_config.md

```diff
@@ -86,7 +86,9 @@ The following arguments are supported:
 - `subject_name` - (Required String) The name of the subject (in other words, the namespace), representing the subject under which the schema will be registered, for example, `test-subject`.
 
 -> **Note:** If you want to reference the subject that is located in a custom context, use the following naming pattern: `:.contextName:subjectName`. For example, use `subject_name = ":.context1:test-subject"` to reference the subject named `test-subject` in the `context1` context, and use `subject_name = "test-subject"` to reference the subject named `test-subject` in the `default` context.
- 
+
+-> **Note:** To configure config at the context level, affecting all subjects created within that context, use the following naming pattern: `:.contextName:`. For example, to set the global configuration for the `context1` context, use `subject_name = ":.context1:"`.
```

Comment:
The only small nitpick I have is 
>To configure config

to
>To configure configs

or
>To configure a config

Reply:
That's a great catch! I applied this suggestion 👍